cmd/trade.go

```diff
@@ -31,6 +32,8 @@ var tradeCmd = &cobra.Command{
 	Example: tradeExamples,
 }
 
+var logger loggers.BasicLogger
```

Comment:
this should not be a top level variable like this.

the logger should be created inside the function `tradeCmd.Run = func(ccmd *cobra.Command, args []string)`. Eventually the config files will dictate where we want to log which will guide the logger object that is created.

we can start the convention in kelp that the logger variable will always be called `l` (lowercase-L). so we can always log using `l.Infof(...)` as an example. Any objects that themselves hold a logger can use it in their own functions like so: `obj.l.Infof()` instead of `obj.logger.Infof()` or `obj.log.Infof()`. This convention is akin to using `e` for errors in kelp instead of `err` to keep things more succinct.